src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -103,55 +101,61 @@ public AutonModes(
+    Trajectory[] trajectories = new Trajectory[pathName.length];
+    for (int i = 0; i < pathName.length; i++) {
+      Path path =
+          Filesystem.getDeployDirectory()
+              .toPath()
+              .resolve(pathName[i]); // goes to scr/main/deploy/paths
+      try {
+        trajectories[i] = TrajectoryUtil.fromPathweaverJson(path);
+        System.out.println("Success: " + pathName[i] + " created.");
+      } catch (IOException e) {
+        System.out.println("Trajectory: " + pathName[i] + " not created.");
+      }
     }
-    return t;
-  }
-
-  private RamseteCommand getRamseteCommand(Trajectory trajectory) {
-    RamseteCommand ramseteCommand =
-        new RamseteCommand(
-            trajectory,
-            driveBaseSubsystem::getPose,
-            new RamseteController(
-                Constants.ramseteB, Constants.ramseteZeta), // ramsete follower to follow trajectory
-            Constants.driveKinematics,
-            driveBaseSubsystem::acceptWheelSpeeds,
-            driveBaseSubsystem);
-
-    return ramseteCommand;
+    return trajectories;
   }
 
-  private void initializeTrajectories() {
-    taxiTrajectory = getTrajectory(Constants.taxiPath);
-
-    if (allSubsystemsEnabled) {
-      oneBallTrajectory = getTrajectory(Constants.oneBallPath);
-
-      twoBallTrajectory[0] = getTrajectory(Constants.twoBallPath[0]);
-      twoBallTrajectory[1] = getTrajectory(Constants.twoBallPath[1]);
-
-      // threeBallTrajectories
-      // fourBallTrajectories
+  private Command[] getRamseteCommands(Trajectory... trajectories) {
+    Command[] ramseteCommands = new Command[trajectories.length];
+    for (int i = 0; i < trajectories.length; i++) {
+      RamseteCommand r =
+          new RamseteCommand(
+              trajectories[i],
+              driveBaseSubsystem::getPose,
+              new RamseteController(
+                  Constants.ramseteB,
+                  Constants.ramseteZeta), // ramsete follower to follow trajectory
+              Constants.driveKinematics,
+              driveBaseSubsystem::acceptWheelSpeeds,
+              driveBaseSubsystem);
+
+      // first ramsete command needs to have driveBase reset odometry to match that of pathweaver
+      if (i == 0) {
+        ramseteCommands[i] =
+            r.beforeStarting(
+                () -> driveBaseSubsystem.resetOdometry(trajectories[0].getInitialPose()));
+      } else {
+        ramseteCommands[i] = r;
+      }
     }
+
+    return ramseteCommands;
   }
 
   private void initializeRamseteCommands() {
-    taxiRamseteCommand = getRamseteCommand(taxiTrajectory);
+    taxiRamseteCommand =
+        getRamseteCommands(getTrajectories(Constants.Auton.TAXI.getPaths()))[
+            0]; // only has one trajectory so only one ramsete command (first element of array)
 
-    if (allSubsystemsEnabled) {
-      oneBallRamseteCommand = getRamseteCommand(oneBallTrajectory);
+    if (shooterEnabled) {
```

Comment:
Related to previous comment on 95. We probably don't need this for competition, so ideally we'd get rid of this. At minimum, it should be easier for the user to remember to set and disable (tied to modular subsystem or dashboard).